graph/iterator/lines_map.go

```diff
@@ -0,0 +1,140 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !safe
+// +build !safe
+
+package iterator
+
+import "gonum.org/v1/gonum/graph"
+
+// Lines implements the graph.Lines interfaces.
+// The iteration order of Lines is randomized.
+type Lines struct {
+	lines int
+	iter  *mapIter
+	pos   int
+	curr  graph.Line
+}
+
+// NewLines returns a Lines initialized with the provided lines, a
+// map of line IDs to graph.Lines. No check is made that the keys
+// match the graph.Line IDs, and the map keys are not used.
+//
+// Behavior of the Lines is unspecified if lines is mutated after
+// the call the NewLines.
```

Comment:
s/the/to throughout

Reply:
Thanks. Fixed in other files not touched here too.